### PR TITLE
ref(replays): Streamline the DeadRage click cards component

### PR DESCRIPTION
**Pull request by ryan953**
Depends on https://github.com/getsentry/sentry/pull/56133

### CODE REVIEW

File: static/app/views/replays/list/replaysErroneousDeadRageCards.tsx [hunk 1]
```diff
@@ -191,37 +165,39 @@ function CardTable({
         sort={undefined}
         visibleColumns={visibleColumns}
         saveLocation
-        gridRows={'auto ' + gridRows}
+        gridRows={`auto ${repeat(' 1fr', rows)}`}
```
Comment (billyvg) [inline static/app/views/replays/list/replaysErroneousDeadRageCards.tsx:168]: Can you use CSS [`repeat()`](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat) here?

```suggestion
        gridRows={`auto repeat(${rows}, 1fr)`}
```

File: static/app/views/replays/list/replaysErroneousDeadRageCards.tsx [hunk 2]
```diff
@@ -191,37 +165,39 @@ function CardTable({
...
+        browserHistory.push({
+          pathname: location.pathname,
+          query: {
+            ...location.query,
+            cursor: undefined,
+            query: isActive ? '' : eventView.query,
+            sort: isActive ? '' : eventView.sorts[0].field,
+          },
+        });
+      }}
```
Comment (billyvg) [inline static/app/views/replays/list/replaysErroneousDeadRageCards.tsx:193]: m: This should probably be a link (I don't remember the component for a link that looks like a button) instead?

File: static/app/views/replays/list/replaysErroneousDeadRageCards.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/views/replays/list/replaysErroneousDeadRageCards.tsx:168]: ooh, i didn't try!

File: static/app/views/replays/list/replaysErroneousDeadRageCards.tsx (hunk 2 above)
Comment (ryan953) [reply static/app/views/replays/list/replaysErroneousDeadRageCards.tsx:193]: 👍 LinkButton

**Review by billyvg — APPROVED** [review]

**Pull request merged**